packages/suite/src/hooks/wallet/useSendForm.ts

```diff
@@ -273,7 +277,7 @@ export const useSendForm = (props: UseSendFormProps): SendContextValues => {
                 dispatch(goto('wallet-index', { preserveParams: true }));
             }
         }
-    }, [getValues, composedLevels, dispatch, resetContext]);
+    }, [getValues, composedLevels, dispatch, resetContext, props.selectedAccount.account]);
```

Comment:
You already have this destructured above. Can be used here

```suggestion
    }, [getValues, composedLevels, dispatch, resetContext, account]);
```

Reply:
The destructuration operator is in a different function.